sycl/source/detail/graph_impl.cpp

```diff
@@ -378,83 +380,96 @@ sycl::event exec_graph_impl::enqueue(
     NewEvent->setStateIncomplete();
     return NewEvent;
   });
-#if SYCL_EXT_ONEAPI_GRAPH
-  auto NewEvent = CreateNewEvent();
-  RT::PiEvent *OutEvent = &NewEvent->getHandleRef();
-  auto CommandBuffer = MPiCommandBuffers[Queue->get_device()];
-
-  // If we have no requirements for accessors for the command buffer, enqueue it
-  // directly
-  if (MRequirements.empty()) {
-    pi_result Res =
-        Queue->getPlugin()
-            .call_nocheck<sycl::detail::PiApiKind::piextEnqueueCommandBuffer>(
-                CommandBuffer, Queue->getHandleRef(), RawEvents.size(),
-                RawEvents.empty() ? nullptr : &RawEvents[0], OutEvent);
-    if (Res != pi_result::PI_SUCCESS) {
-      throw sycl::exception(
-          errc::event, "Failed to enqueue event for command buffer submission");
-    }
-  } else {
-    std::unique_ptr<sycl::detail::CG> CommandGroup =
-        std::make_unique<sycl::detail::CGExecCommandBuffer>(CommandBuffer,
-                                                            MRequirements);
 
```

Comment:
Minor note for future PRs but I think this blank line here has made the diff for this section quite difficult to parse. I've encountered this issue before and found it can be worth checking for things like this to see if they can be removed to make things easier to review.

Reply:
This diff is definitely noisier than I thought it would be. I tried removing the blank line, but didn't seem to help much. I'll bear this in mind to help similar situations in future.